libmscore/beam.cpp

```diff
@@ -62,7 +62,6 @@ Beam::Beam(Score* s)
       _direction       = Direction::AUTO;
       _up              = true;
       _distribute      = false;
-      _noSlope         = false;
```

Comment:
Where does the "overwriting" happen? I'm curious because here I see normal value initialization, perhaps we need to find out the "root" of this issue somwhere

Reply:
<code>_noSlopen</code> is a style parameters and set in <code>setProperty()</code> which is called by <code>initElementStyle()</code>, just a few line before this one. So, <code>_noSlope</code> is set as a style parameter and set to <code>false</code> here.
But make sure, <code>_noSlope</code> is always initialised, it might be better to move <code>initElementStyle()</code> to **after** the initialisation of the parameters.

Reply:
BTW, it seems more elements might suffer this same issue, style parameter overwritten by initialisation. Is it better to check and extend this PR or create a new one?

Reply:
PVS Studio reported that member value not initialized by default. So I added this code, like others.
I agree, better move initElementStyle() to after the initialisation of the parameters.

Reply:
Let's add the write call ordering here and then I'll merge it
If there is the same problem in other elements, then I'd solve it in a separate PR

Reply:
@igorkorsukov Thanks for the explanation.
Update in on its way and after that, I'll check the other elements and create a separate PR for that.